release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -3292,14 +3331,7 @@ Issued: 5 February 2025
 
 {product-title} release {product-version}.15 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:0876[RHSA-2025:0876] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:0878[RHSA-2025:0878] advisory.
 
-Space precluded documenting all of the container images for this release in the advisory.
-
-You can view the container images in this release by running the following command:
-
-[source,terminal]
-----
-$ oc adm release info 4.17.15 --pullspecs
-----
+Space precluded documenting all of the containe (link:https://issues.redhat.com/browse/OCPBUGS-54534[*OCPBUGS-54534*])
```

Comment:
HI @tedaveryredhat . This PR will need to be merged today. Can you return this section to it's original for 4.17.15?

Reply:
Ideally z-streams docs should go out the same day as the last advisory to hit SHIPPED_LIVE status.